internal/commands/login/inputs_test.go

```diff
@@ -167,9 +177,36 @@ func TestLoginInputs(t *testing.T) {
 				assert.Equal(t, "password", i.Password)
 			},
 		},
+		{
+			description:    "Should not open browser if no-browser flag is set",
+			inputs:         inputs{AuthType: authTypeCloud, NoBrowser: true},
+			prepareProfile: func(p *user.Profile) {},
+			procedure: func(c *expect.Console) {
+				c.ExpectString("API Key")
+				c.SendLine("username")
+				c.ExpectString("Private API Key")
+				c.SendLine("password")
+				c.ExpectEOF()
+			},
+			test: func(t *testing.T, i inputs) {
+				assert.Equal(t, "username", i.PublicAPIKey)
+				assert.Equal(t, "password", i.PrivateAPIKey)
+			},
+			expectBrowser: false,
```

Comment:
[style/nit] I tend to leave zero values out of the `tc` declarations for brevity/ease-of-reading